skycoin/skywire
# Add `gocyclo` to local and CI lint

**Did you run `make format && make check`?** Yes

**Fixes** #1865 

**Changes:**	
- add `gocyclo` to checking, both on local and CI process

**How to test this PR:**
- use `make check` (or `make check-windows`) for seeing `gocyclo` output

**Note**: Currently, we use `-over 14` flag, as used in Go Report Card, but now ignore issues in codebase, and should handle later as describe here: #1905
